dissect/target/plugins/apps/browser/chromium.py

```diff
@@ -371,6 +378,119 @@ def extensions(self, browser_name: Optional[str] = None) -> Iterator[BrowserExte
                 except (AttributeError, KeyError) as e:
                     self.target.log.info("No browser extensions found in: %s", json_file, exc_info=e)
 
+    def _get_local_state_key(self, local_state_path: TargetPath, chromium_passwords: list) -> Union[bytes, bool]:
+        """Get the Chromium os_crypt encrypted_key and decrypt the key using DPAPI."""
+
+        if not local_state_path.exists():
+            self.target.log.warning(f"File {local_state_path} does not exist.")
+            return False
```

Comment:
Probably better to make this function return `Optional[bytes]` and return `None` if the key is not available.